in_command_test.go

```diff
@@ -464,9 +464,9 @@ var _ = Describe("In Command", func() {
 		Context("which has a tag", func() {
 			BeforeEach(func() {
 				githubClient.GetReleaseReturns(buildRelease(1, "v0.35.0", true), nil)
-				githubClient.GetRefReturns(buildTagRef("v0.35.0", "f28085a4a8f744da83411f5e09fd7b1709149eee"), nil)
 
 				inRequest.Version = &resource.Version{ID: "1"}
+				inRequest.Source.Drafts = true
```

Comment:
This change would no longer be necessary with the approach mentioned above, since we already set `draft` to `true` on the release (third argument to `buildRelease`).